short/index.html

```diff
@@ -298,6 +298,35 @@
 </section>
 
 <section data-markdown>
+**Exercise**
+
+```python
+def test_overlay_no_overlap():
+    rect1 = Rectangle(0, 0, 1, 1)
```

Comment:
I think these tests will be a lot more comprehensible to people who aren't fluent in Python if they're accompanied by Cartesian grid diagrams showing the test cases.